waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMappingImpl.java

```diff
@@ -88,7 +88,15 @@ public MetaStoreFilterHook getMetastoreFilter() {
 
   @Override
   public Table transformOutboundTable(Table table) {
-    table.setDbName(metaStoreMapping.transformOutboundDatabaseName(table.getDbName()));
+    String originalDatabaseName = table.getDbName();
+    String databaseName = metaStoreMapping.transformOutboundDatabaseName(originalDatabaseName);
+    table.setDbName(databaseName);
+    if (databaseName.equalsIgnoreCase(originalDatabaseName)) {
+      // Skip all the view parsing if nothing is going to change, the parsing is not without problems and we can't catch
```

Comment:
is it worth keeping the code on line 123 since that will throw an error still?

Reply:
the original can't have the same string as the extended cause it also needs the colon. if we add it will it still throw an error?

Reply:
I return the table in this if so we'll not even try to parse.

Reply:
yes, but if the database name is different it will still throw an error

Reply:
No because I skip parsing in the parser class for Presto markers. The if in here is to ensure other crazy markers don't also through errors. The code here is to basically get back to same support as in pre WD 3.6.0. WD 3.7.0 introduced the problems.
If we really have to support Presto I got this follow up: https://github.com/HotelsDotCom/waggle-dance/issues/212

Reply:
OK, thanks for creating that issue. So for now we'll return the Presto view "as is" but that could mean it won't actually work since it doesn't have the correct prefixes and that's what #212 is all about.

Reply:
correct this is fixing and reverting old behaviour. Make it work again if there are no prefixes to remove.